lib/puppet/configurer/plugin_handler.rb

```diff
@@ -35,7 +35,7 @@ def download_plugins(environment)
         "locales",
         Puppet[:localedest],
         Puppet[:localesource],
-        Puppet[:localeignore],
+        Puppet[:pluginsignore] + " *.pot config.yaml",
```

Comment:
AFAICT `localeignore` is not a setting, and this was a mistake made when the downloader factory was merged into the plugin handler. I can make this a separate maint commit instead of part of the merge commit if that's preferred